docs/excel/custom-functions-overview.md

```diff
@@ -61,6 +61,12 @@ function add(first, second){
  CustomFunctions.associate("ADD", add);
 ```
 
+Note that the functions.html file, which governs the loading of the custom functions runtime, should contain the current CDN for custom functions. If you have prepared your project using the most current version of yo office, this CDN will already be present, but if you are retrofitting a previous custom function from March 2019 or earlier, you may need to copy in the code below to the functions.html page.
```

Comment:
Consider breaking the last sentence up. Something like:
"Projects prepared with the current version of yo office reference the CDN, If you are retrofitting a previous custom function project from March 2019 or earlier, you need to copy in the code below to the **functions.html** page."